# command_sequence index
# [1] command:
#   1. takeoff = Taking off sequence
#   2. hold = Wait for certain time
#   3. goto = Move to location
#   4. goto_velocity = Move to location with velocity control
#   5. external = Wait for external command
#   6. land = Landing sequence
# [2] to wait before the next command:
#   1. conc = Go to the next command without waiting for this
#   2. wait = Wait for this command
# [3] drone number involved
#   1. "all"
#   2. "cfX" = split the cfs by underscore etc "cf1_cf2_cf3"
# [4] duration (only applicable to hold) in ms, if nothing leave empty ""
# [5] pose in XYZ "1 1 1", if nothing leave empty ""

# command_sequence: [""]
queue_size: 20
# add in the environment file name here
# environment_file: ""
environment_file: "maze.yaml"
trajectory_parameters:
  max_velocity: 0.5
  reached_threshold: 0.175
  takeoff_land_velocity: 0.5
  takeoff_height: 1.0
  planning_rate: 8.0
  communication_radius: 5.0
  protected_zone: 0.1
  planning_horizon_scale: 3.0
  height_range: [0.5, 2.0]
april_tag_parameters:
  # 35 degs pointing downwards
  camera_rotation: [ 0, 0.3007058, 0, 0.953717 ] # x,y,z,w
  time_threshold: 0.100
  observation_threshold: 2.00
  observation_limit: 2
  is_z_out: true
rviz:
  text:
    scale_factor: 32.0
sim:
  hfov: 75.0 # deg
  vfov: 50.0 # deg
  observation:
    clamp_distance: 1.5